docs/references/ruby/rails.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Ruby on Rails integration
+description: The Clerk Ruby SDK provides a Rack middleware and a gem to integrate Clerk into your Rails app.
+---
+
+The Clerk Ruby SDK provides a seamless integration with Ruby on Rails through a Rack middleware and dedicated Rails helpers. When you add the Clerk gem to your Rails application, the middleware is automatically included in your application's middleware stack.
+
+<Steps>
+  ## Install `clerk-sdk-ruby`
+
+  <Include src="_partials/ruby/installation" />
+
+  ## Configuration
+
+  <Include src="_partials/ruby/configuration" />
+</Steps>
+
+## Example: Access the `clerk` object
+
+To access Clerk's authentication functionality in your controllers, include the `Clerk::Authenticatable` concern. This gives your controller and views access to the `clerk` object. The `clerk` object provides access to the [Ruby SDK's available methods](/docs/references/ruby/overview#available-methods).
+
+{/* TODO: I don't think "This gives your controller and views access to the `clerk` object." is correct. Because that's what the middleware does. So what does the `Clerk::Authenticatable` concern specifically do? */}
```

Comment:
```suggestion
To access Clerk's authentication functionality in your controllers, include the `Clerk::Authenticatable` concern. This gives your controller and views access to the `clerk` helper, allowing for you to use the sdk and retrieve all current session claims via, `.user`, `.organization`, and more.
```

Reply:
The middleware does the heavy-lifting of parsing out everything and making it available in the environment. `Clerk::Authenticatable` makes it so that that information is available is available via the `clerk` helper.